### ISSUE TITLE
Bug: Scroll wheel does not work on Win7

### ISSUE THREAD

**Issue by 151083** [Kyusung4698/PoE-Overlay#319]
In the evaluate options i have chosen alchemy, exalted and chaos as evaluate options. Whenever i try to change switch the price for different currency with mouse wheel nothing happens.

**Comment by platun4** [comment]
I have same problem. Scrolling does nothing, zooms in and out game camera if ui over ground.

**Comment by 151083** [comment]
@Kyusung4698

**Comment by jnitis** [comment]
> zooms in and out game camera if ui over ground.

This would appear to be unintentional behavior, can we get this fixed please?

Issue is if mouse is over game terrain (ie, not stash/inventory/etc.) ctrl-scroll zooms the main game window in/out in addition to scrolling the stash tabs.  I have to either fix my zoom level every time I access my stash or first ensure my mouse is not over the terrain before I scroll tabs.  I checked the old poe trade macro to see if it did this and it does not, perhaps due to a different implementation here.

Thanks!

**Comment by Kyusung4698** [comment]
Stash navigation zooming won't be fixed. See: #146 

There is an issue with scrolling on Win7 which probably won't be fixed. It's buried deep in Electron and i don't think they'll work on it - as Win7 has reached End of Support.

**Comment by kittensays** [comment]
> There is an issue with scrolling on Win7 which probably won't be fixed. 
Could you add some kind of arrows to click on it or keybind to do it with keyboard then? if it possible of course.

**Comment by Kyusung4698** [comment]
## 0.6.11 (2020-03-24)

- fix scroll wheel not working on win7 (#319)

**Comment by kittensays** [comment]
> Looking for feedback! I could only tested it on a VM.

For me it's working. Thanks.

**Comment by Kyusung4698** [comment]
> > Looking for feedback! I could only tested it on a VM.
> 
> For me it's working. Thanks.

Awesome! Thanks for your feedback 🚀 I'll close this issue for now.

**Comment by Kessus** [comment]
For me it's still not working... Are there any requirements for it? Maybe I don't have something you used to fix it?

**Comment by Kyusung4698** [comment]
> For me it's still not working... Are there any requirements for it? Maybe I don't have something you used to fix it?

For win7 you'd need to enable keyboard support at the tray menu. This is currently a experimental feature but it fixes the focus problem of win7.

**Comment by Kessus** [comment]
It doesn't focus on its own. I need to click the value with my scroll wheel and drag it to the side before it starts working properly. A circular cursor with two arrows appears. It does nothing by itself (however its appearance suggests that it should also scroll the values) but it enables the scroll to work properly with either only that particular mod or all mods of this search. The first time I tried it, only one mod reacted to the scroll. When I did it to the second mod, all of them started catching the scroll and working but only for that one search. For another one I had to do it again.
